powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationHistoryEntity.java

```diff
@@ -71,14 +87,21 @@ public ActivationHistoryEntity() {
      * @param id                Signature audit item record ID.
      * @param activation        Associated activation, or null of no related activation was found.
      * @param activationStatus  Activation status at the time of signature computation attempt.
-     * @param timestampCreated  Created timestapm.
+     * @param timestampCreated  Created timestamp.
+     * @param version           Activation version.
      */
-    public ActivationHistoryEntity(Long id, ActivationRecordEntity activation, ActivationStatus activationStatus, Date timestampCreated) {
-        super();
+    public ActivationHistoryEntity(
```

Comment:
Jak moc řešíte zpětnou kompatibilitu, co je pro vás API? Jen REST a SOAP? Byl jsem zvyklý, že změnu konstruktoru entity si nemůžu dovolit, protože to mohly používat extenze.

Reply:
The constructor can change, we mainly care for these concerns related to backward compatibility:
- REST API compatibility
- SOAP API compatibility
- Error codes
- Database compatibility
- Java compatibility

The extensions are mostly done using `powerauth-restful-integration` or `enrollment-server` projects.